factcast-store-pgsql/src/main/resources/db/changelog/factcast/issue1161/storage-setting.sql

```diff
@@ -0,0 +1,19 @@
+alter table fact
+    alter column header set storage MAIN;
+alter table fact
+    alter column payload set storage EXTENDED;
```

Comment:
Interesting, but I don't fully get it. The doc says on high-level _A table that has columns with potentially large entries will have an associated TOAST table, which is used for out-of-line storage of field values that are too large to keep in the table rows proper._ I would not consider an average fact a large entry - what benefit does using TOAST bring for FactCast?

Reply:
when scanning, it is important to have the header inline with the row in case of filtering, whereas - if a row should exceed 8k - it should be the payload being stored out of band, not the header.
The sole purpose of this is i/o cache efficiency.